hadoop-crypto-api/build.gradle

```diff
@@ -7,4 +7,5 @@ dependencies {
     compile "org.slf4j:slf4j-api"
     compile "com.fasterxml.jackson.core:jackson-databind"
     compile "com.google.guava:guava"
+    compile "org.apache.hadoop:hadoop-common"
```

Comment:
I think that may introduce a legal issue: https://github.com/apache/commons-crypto/blob/master/pom.xml#L49